### PR TITLE
Add JSON for download return

### PR DESCRIPTION
**Pull request by AlbertLlanos**
Json for Download returns on CONVERT_SERVICES

### CODE REVIEW

File: CONVERT_SERVICES/src/controller/apis/endpointconverter.py [hunk 1]
```diff
@@ -33,4 +33,11 @@ def getRequest(self):
...
-        return ( r'http://127.0.0.1:5000/downloader/' + os.path.join(dir_output, file_name))
\ No newline at end of file
+        path = r'http://127.0.0.1:5000/downloader/' + os.path.join(dir_output, file_name)
+        download_path = {
+            'dir_output': dir_output,
+            'file_name': file_name,
+            'os_path': os.path.join(dir_output, file_name),
+            'path': path
+        }
+        return (download_path)
```
Comment (FreddyClaros) [inline CONVERT_SERVICES/src/controller/apis/endpointconverter.py:43]: add a space at the end please

File: CONVERT_SERVICES/src/controller/apis/endpointconverter.py [hunk 2]
```diff
@@ -33,4 +33,11 @@ def getRequest(self):
         return self.request
 
     def Send_File(self, dir_output, file_name):
```
Comment (FreddyClaros) [inline CONVERT_SERVICES/src/controller/apis/endpointconverter.py:35]: here if yo can change the function name to send_file in lowercase, uppercase it's for classes

**Review by Amilcar-B — APPROVED** [review]

**Comment by daylerVMT** [comment]
the code is updata in freddys branch I closed this feature

**Review by daylerVMT — COMMENTED** [review]
closed branch now implement in branch freddy

**Pull request closed without merging**